dynamic-rapidpro-workspace-settings/test/index.spec.js

```diff
@@ -0,0 +1,94 @@
+const { expect } = require('chai');
+const path = require('path');
+const sinon = require('sinon');
+const utils = require('../utils');
+const secrets = require('./env');
+const settings = require('./app_settings.json');
+const flows = require('./flows');
+const fs = require('fs');
+const axios = require('axios').default;
+
+let sandbox = sinon.createSandbox();
+
+describe(`rapidpro action test suite`, () => {
+  const mockedAxiosResponse = {
+    data: {},
+    status: 200,
```

Comment:
How would the code behave with a non-2xx response?

Reply:
Added an extra test.